crates/burn-import/src/onnx/to_burn.rs

```diff
@@ -1082,12 +1073,27 @@ impl ParsedOnnxGraph {
         UnaryNode::exp(input, output)
     }
 
-    fn expand_conversion(node: Node) -> ExpandNode {
+    fn expand_conversion(mut node: Node) -> ExpandNode {
         let input = TensorType::from(node.inputs.first().unwrap());
-        let output = TensorType::from(node.outputs.first().unwrap());
         let shape = expand_config(&node);
 
-        ExpandNode::new(input, output, shape)
+        // dim_interference left the dim at zero, so it needs to be filled before converting to TensorType:
```

Comment:
I know it's just a comment, but "dim interference" should be "dim inference" 😄

Reply:
Fixed together with the rebase